src/ft2font.h

```diff
@@ -18,8 +18,8 @@ extern "C" {
 /*
  By definition, FT_FIXED as 2 16bit values stored in a single long.
  */
-#define FIXED_MAJOR(val) (long)((val & 0xffff000) >> 16)
-#define FIXED_MINOR(val) (long)(val & 0xffff)
+#define FIXED_MAJOR(val) (unsigned short)((val & 0xffff000) >> 16)
```

Comment:
Oh,  so they're what the freestype docs call something like FT_MAJOR and FT_MINOR?